cves/kernel/CVE-2021-3609.yml

```diff
@@ -282,8 +299,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The fix to this vulnerability involved synchronizing different operations,
+    so I assume they were communicating in some fashion.
```

Comment:
Were they communicating across processes or were they synchronizing threads? 

Also, saying "I assume" negates some of your credibility, double-check the facts and rewrite this with some certainty.